crates/qt-build-utils/src/lib.rs

```diff
@@ -350,7 +381,7 @@ impl QtBuild {
         qt_module: &str,
     ) -> String {
         for arch in ["", "_arm64-v8a", "_armeabi-v7a", "_x86", "_x86_64"] {
-            let prl_path = format!(
+            let prl_path: String = format!(
```

Comment:
do we need this type? doesn't `format!(...)` always go to String?

Reply:
Force of habit from C++, typing all, sorry
```suggestion
            let prl_path = format!(
```